{{ cookiecutter.package_name|slugify }}/Dockerfile

```diff
@@ -35,32 +56,13 @@ RUN --mount=type=cache,id=apt-cache,target=/var/cache/apt/ \
     echo 'eval "$(starship init zsh)"' >> ~/.zshrc && \
     zsh -c 'source ~/.zshrc'
 
-# Install the development Python environment.
-COPY .pre-commit-config.yaml poetry.lock* pyproject.toml /app/
-RUN --mount=type=cache,id=poetry,target=/root/.cache/ \
-    {%- if cookiecutter.private_package_repository_name %}
-    --mount=type=secret,id=poetry_auth,target=/root/.config/pypoetry/auth.toml \
-    {%- endif %}
-    mkdir -p src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/ && touch src/{{ cookiecutter.package_name|slugify|replace("-", "_") }}/__init__.py && touch README.md && \
-    poetry install --no-interaction && \
-    mkdir -p /var/lib/poetry/ && cp poetry.lock /var/lib/poetry/ && \
+# Persist output generated during docker build so that we can restore it in the dev container.
+RUN mkdir -p /var/lib/poetry/ && cp poetry.lock /var/lib/poetry/ && \
```

Comment:
The `.pre-commit-config.yaml` should be copied here, because we'll use it to install pre-commit on line 61.

Reply:
Replied to the original comment

Reply:
Resolved original comment 🙂